pkg/apis/v1alpha2/aksnodeclass.go

```diff
@@ -53,8 +53,8 @@ type AKSNodeClassSpec struct {
 	// +optional
 	Kubelet *KubeletConfiguration `json:"kubelet,omitempty" hash:"ignore"`
 	// MaxPods is an override for the maximum number of pods that can run on a worker node instance.
-	// +kubebuilder:validation:Minimum:=0
-	// +kubebuilder:default=250
+	// +kubebuilder:validation:Minimum:=10
+	// +kubebuilder:validation:Maximum:=250
```

Comment:
Where does this limitation come from? Please add a comment (if it is real)

Reply:
10 is the aks minimum, and 250 is the aks maximum.